addons/config/templates/csi.tanzu.vmware.com/v1alpha1/azurediskcsiconfig.yaml

```diff
@@ -0,0 +1,13 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: csi.tanzu.vmware.com/v1alpha1
+kind: AzureDiskCSIConfig
+metadata:
+  name: #@ data.values.TKR_VERSION
+  namespace: #@ data.values.GLOBAL_NAMESPACE
+  annotations:
+    tkg.tanzu.vmware.com/template-config: "true"
+spec:
+  azureDiskCSIDriver:
+    namespace: tkg-system
```

Comment:
kube-system

Reply:
Thanks, will add a fix patch on the new branch